core/src/test/java/org/testcontainers/junit/OutputStreamWithTTYTest.java

```diff
@@ -0,0 +1,94 @@
+package org.testcontainers.junit;
+
+import com.github.dockerjava.api.command.CreateContainerCmd;
+import org.junit.Rule;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.output.OutputFrame;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+import org.testcontainers.containers.output.ToStringConsumer;
+import org.testcontainers.containers.output.WaitingConsumer;
+
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import java.util.function.Consumer;
+
+import static org.rnorth.visibleassertions.VisibleAssertions.*;
+import static org.testcontainers.containers.output.OutputFrame.OutputType.STDOUT;
+
+public class OutputStreamWithTTYTest {
+
+    @Rule
+    public GenericContainer container = new GenericContainer("alpine:3.2")
```

Comment:
if you change to `GenericContainer<>`, you can remove `((CreateContainerCmd)command)` cast